receiver/awscontainerinsightreceiver/internal/neuron/neuron_monitor_scraper_config.go

```diff
@@ -72,10 +90,17 @@ func GetNeuronMetricRelabelConfigs(hostinfo prometheusscraper.HostInfoProvider)
 		// relabel looks up an existing label then creates another label with given key (TargetLabel) and value (static)
 		{
 			SourceLabels: model.LabelNames{"instance_id"},
-			TargetLabel:  "ClusterName",
+			TargetLabel:  ci.ClusterNameKey,
 			Regex:        relabel.MustNewRegexp("(.*)"),
 			Replacement:  hostinfo.GetClusterName(),
 			Action:       relabel.Replace,
 		},
+		{
+			SourceLabels: model.LabelNames{"instance_id"},
+			TargetLabel:  ci.NodeNameKey,
```

Comment:
Good catch.  I see there are other tags in [cadvisor](https://github.com/amazon-contributing/opentelemetry-collector-contrib/blob/ca8e105c7708dd587630b06e86d9b0c815ca34c3/receiver/awscontainerinsightreceiver/internal/cadvisor/cadvisor_linux.go#L259) that we're also not populating - Version, AutoScalingGroupName, maybe others.  Do we need to add those too?